pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -1203,9 +1171,56 @@ func (o *MustGatherOptions) newPod(node, image string, hasMaster bool, affinity
 					Operator: "Exists",
 				},
 			},
-			Affinity: affinity,
 		},
 	}
+	if update != nil {
+		update(ret)
+	}
+	return ret
+}
+
+// newPod creates a pod with 2 containers with a shared volume mount:
+// - gather: init containers that run gather command
+// - copy: no-op container we can exec into
+func (o *MustGatherOptions) newPod(node, image string, hasMaster bool, affinity *corev1.Affinity) *corev1.Pod {
+	nodeSelector := map[string]string{
+		corev1.LabelOSStable: "linux",
+	}
```

Comment:
Shouldn't this be taken from the default pod, or at least merged somehow? Otherwise the values defined in the default pod are simply dropped.